packages/consensus/src/types/transaction.cairo

```diff
@@ -127,6 +127,25 @@ pub struct TxOut {
     pub cached: bool,
 }
 
+///
+/// Custom implementation of the Hash trait for TxOut removed cached field.
+///
+impl TxOutHash<S, impl SHashState: HashStateTrait<S>, +Drop<S>> of Hash<TxOut, S, SHashState> {
+    #[inline(always)]
```

Comment:
Better avoid inline attributes

Reply:
I copy pasted exactly as Hash macros is being unwrap. Feels like if it is safe there it'll be safe here. Also the logic is quite straightforward, so should be fine. What is your concern?

Reply:
https://book.cairo-lang.org/ch11-06-inlining-in-cairo.html#the-inline-attribute

It doesn't necessarily reduce code size, but would complicate debugging/profiling